modules/snooker/src/main/ts/ephox/snooker/resize/ColumnSizes.ts

```diff
@@ -48,12 +48,12 @@ const getDeduced = function (deduced: Optional<number>) {
   return deduced.map(function (d) { return d + 'px'; }).getOr('');
 };
 
-const getRawWidths = function (warehouse: Warehouse, direction: BarPositions<ColInfo>, tableSize: TableSize) { // Warning, changed signature!
-  return getWidthFrom(warehouse, direction, getRawW, getDeduced, tableSize);
+const getRawWidths = function (warehouse: Warehouse, tableSize: TableSize) { // Warning, changed signature!
```

Comment:
That comment seems like something that's not going to age well with the codebase

Reply:
...I'm going to delete that. Thanks!